### ISSUE TITLE
Add token parameter to LangfuseConnector

### ISSUE THREAD

**Issue by Emil-io** [deepset-ai/haystack-core-integrations#1235]
**Is your feature request related to a problem? Please describe.**
Currently haystack supports Secrets. This is not consistent, with the LangfuseConnector. For this component, the keys and other settings (like host, ...) are set as environment variables. This does not align with they way one configures the OpenAIGenerator as an example.

**Describe the solution you'd like**
Update the init of the Langfuse Connector, so that it has all the parameters, which have to be set through environment variables as of now (auth keys, host, ...)

**Comment by julian-risch** [comment]
Hi @Emil-io just to double check: The three parameters we are talking about are LANGFUSE_SECRET_KEY, LANGFUSE_PUBLIC_KEY and LANGFUSE_HOST, correct? Any others? I assume LANGFUSE_RELEASE, LANGFUSE_THREADS aren't needed.

If it's just those, we should update the init of the LangfuseConnector to include
```python
secret_key: Secret = Secret.from_env_var("LANGFUSE_SECRET_KEY"),
public_key: Secret = Secret.from_env_var("LANGFUSE_PUBLIC_KEY"),
host: Secret = Secret.from_env_var("LANGFUSE_HOST")
```

and then when we initialize Langfuse, we should pass the parameters:

```python
self.tracer = LangfuseTracer(tracer=Langfuse(secret_key=secret_key, public_key=public_key, host=host), name=name, public=public)
```

based on https://langfuse.com/docs/sdk/python/low-level-sdk

**Comment by Emil-io** [comment]
Exactly. Maybe also a parameter, through which one can control this:

> In addition, you need to set the HAYSTACK_CONTENT_TRACING_ENABLED environment variable to true in order to enable Haystack tracing in your pipeline. (https://docs.haystack.deepset.ai/reference/integrations-langfuse)

**Comment by vblagoje** [comment]
I think we should perhaps only make these params handled by Secret mechanism. Handling other via Secret would make a precedent. 
secret_key: Secret = Secret.from_env_var("LANGFUSE_SECRET_KEY"),
public_key: Secret = Secret.from_env_var("LANGFUSE_PUBLIC_KEY"),

Perhaps we should add LANGFUSE_HOST and HAYSTACK_CONTENT_TRACING_ENABLED as regular init params that are serialized but adding them as Secret doesn't make much sense imho. 
Feedback needed @julian-risch
